dags/main_summary.py

```diff
@@ -363,16 +363,16 @@
 
 clients_last_seen = bigquery_etl_query(
     task_id="clients_last_seen",
-    destination_table="clients_last_seen_v1",
+    destination_table="clients_last_seen_raw_v1",
     owner="relud@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
+    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
     depends_on_past=True,
     start_date=datetime(2019, 4, 15),
     dag=dag)
 
 clients_last_seen_export = SubDagOperator(
     subdag=export_to_parquet(
-        table="clients_last_seen_v1",
+        table="clients_last_seen_raw_v1",
```

Comment:
@relud Do you know if it's possible for export_to_parquet to export a view rather than a table? I assume at least that it would more expensive since exporting a table can rely on the Copy API, but I think it would be worth eating the query cost to export the view that includes `days_since_seen` rather than just the bit arrays which would be difficult to work with in Athena, etc. But I'm not sure if the connector can handle targeting a view.

Reply:
i haven't tried, let me check

Reply:
> java.lang.RuntimeException: Table type VIEW is not supported

Nope. I can make a utility function that creates a subdag to export a day from a view, or we could make a view in athena. I favor the latter.

Reply:
I forgot that Athena supports views now. If that's feasible, I agree that's a better option. Hopefully, we can make the view union the existing clients_last_seen_v1 data with the new clients_last_seen_raw_v1 data so that we don't have to backfill.

Reply:
I just confirmed that we can do the position calculation as `log2(bitwise_and(16, -16))` in athena, so that's good.

Reply:
I'll coordinate with @robotblake about how best to make these changes.

Reply:
It looks like redash does not display Presto/Athena views in the schema browser. I tried creating a view and querying it, which worked fine, but users would not be able to discover the view on their own.